ring-core/test/ring/middleware/test/file.clj

```diff
@@ -93,3 +93,52 @@
       (is (= 200 status))
       (is (= (into #{} (keys headers)) #{"Content-Length" "Last-Modified"}))
       (is (= foo-html body)))))
+
+(deftest test-wrap-file-with-prefer-handler-option
+  (let [processable-file-uri (str "/" (.getName (File. ^String public-dir "bars.txt")))
+        handler (fn handler
+                  ([request]
+                   (if (= (:uri request) processable-file-uri)
+                     {:status 200
+                      :body   "I'm a blog"}
+                     {:status 404 :body "don't know how to process this file"}))
+
+                  ([request respond raise] ;;async handler
```

Comment:
Can you remove this `;;async handler` comment? Comments should explain "why" rather than "what".